styles/components/alert.module.scss

```diff
@@ -0,0 +1,123 @@
+.popUpWrapper {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(68, 68, 68, 0.5);
+    margin: 0;
+    padding: 0;
+    opacity: 0;
+    z-index: 50;
+    width: 100%;
+    height: 100%;
+    visibility: hidden;
+    transform: scale(1.1);
+    transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+}
+
+.popUp {
+    box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    opacity: 1;
+    border-radius: 10px;
+    //z-index: 999;
+    overflow: hidden;
+    margin: 0;
+    width: 300px;
+    max-width: 100%;
+    height: 200px;
+    max-height: 100%;
+    transform: translate(-50%, -50%);
+    background-color: white;
+    color: #000;
```

Comment:
Stick to a single naming convention: use either `white` and `black` or `#fff` and `#000`